dm/syncer/syncer.go

```diff
@@ -2140,6 +2187,42 @@ func (s *Syncer) Run(ctx context.Context) (err error) {
 	}
 }
 
+func (s *Syncer) checkAndSetSafeModeByBinlogTS(firstBinlogTS int64) error {
```

Comment:
this function is more like initializing, no check or set safe-mode

Reply:
rename to `initSafeModeExitTS `